### DISCUSSION TITLE
Error !!!!!!!!

### DISCUSSION THREAD

**Discussion by ArbabKhan-sudo** [JSBSim-Team/jsbsim#1162] [Q&A]
I am getting this error, I am using pre-built F-35B jsbsim.xml model to calculate Gamma angle & G-Force
```
     JSBSim Flight Dynamics Model v1.2.1 Aug  8 2024 19:02:38
            [JSBSim-ML v2.0]

JSBSim startup beginning ...

Attempt to divide by zero in method     FGColumnVector3::operator/(const double scalar),     object 0 , 0 , 0

  A proper axis type has NOT been selected. Check your aerodynamics definition.
Traceback (most recent call last):
  File line 91, in <module>
    observation, reward, done, info = env.step(action)                          
    self.sim.run()                                     
  File "_jsbsim.pyx", line 1435, in jsbsim._jsbsim.FGFDMExec.run
jsbsim._jsbsim.BaseError:   A proper axis type has NOT been selected. Check your aerodynamics definition.
```

**Comment by agodemar** [comment]
@ArbabKhan-sudo I guess this is an error coming from the RL environment you are using. Probably you should double-check the environment settings before throwing an "Error !!!!!!!!" to this forum.

**Comment by seanmcleod** [comment]
@ArbabKhan-sudo do you think this is the most useful subject line for a discussion?

Have you looked into the error message that JSBSim is giving you?

Check the 6 axes in the aerodynamics section, since JSBSim is telling you that one of them isn't valid.

https://github.com/JSBSim-Team/jsbsim/blob/8e4803b2cbae945fd62e84b1aca121c35edf5174/src/models/FGAerodynamics.cpp#L233-L239

https://github.com/JSBSim-Team/jsbsim/blob/8e4803b2cbae945fd62e84b1aca121c35edf5174/src/models/FGAerodynamics.cpp#L279-L285

Can you provide a link to the F-35 model you're using? If you want someone else to also take a look at the aerodynamics section.

**Comment by bcoconni** [comment]
FWIW this error generally means your FDM is lacking an `<aerodynamics>` section. The easiest way to make JSBSim stop complaining is to add an empty aerodynamics section with `<aerodynamics/>` (mind the slash before the closing bracket).
